src/components/SecondaryPanes/Breakpoints/Breakpoints.css

```diff
@@ -34,57 +37,75 @@
 
 .breakpoints-list .breakpoint-heading,
 .breakpoints-list .breakpoint {
-  font-size: 12px;
   color: var(--theme-content-color1);
   position: relative;
-  transition: all 0.25s ease;
   cursor: pointer;
 }
 
 .breakpoints-list .breakpoint-heading,
 .breakpoints-list .breakpoint,
 .breakpoints-exceptions,
 .breakpoints-exceptions-caught {
-  padding: 0.25em 1em;
+  padding-inline-start: 16px;
+  padding-inline-end: 12px;
 }
 
 .breakpoints-exceptions {
-  padding-bottom: 0.5em;
-  padding-top: 0.5em;
+  padding-bottom: 3px;
+  padding-top: 3px;
   user-select: none;
 }
 
 .breakpoints-list .breakpoint {
-  min-height: var(--breakpoint-expression-height);
   overflow: hidden;
+  padding-top: 2px;
+  padding-bottom: 2px;
+}
+
+.breakpoints-list .breakpoint:last-child {
+  margin-bottom: 4px;
 }
 
 .breakpoints-exceptions-caught {
-  padding: 0 1em 0.5em 3em;
-  margin-top: -0.25em;
+  padding-bottom: 3px;
+  padding-top: 3px;
+  padding-inline-start: 36px;
 }
 
 html[dir="rtl"] .breakpoints-exceptions-caught {
   padding: 0 3em 0.5em 1em;
 }
 
+.breakpoints-exceptions-options {
+  padding-top: 4px;
+  padding-bottom: 4px;
+}
+
+.xhr-breakpoints-pane .breakpoints-exceptions-options {
+  border-bottom: 1px solid var(--theme-splitter-color);
+}
+
 .breakpoints-exceptions-options:not(.empty) {
   border-bottom: 1px solid var(--theme-splitter-color);
-  margin-bottom: 3px;
 }
 
 .breakpoints-exceptions input,
 .breakpoints-exceptions-caught input {
   padding-inline-start: 2px;
+  margin-top: 0px;
+  margin-bottom: 0px;
   margin-inline-start: 0;
+  margin-inline-end: 2px;
   vertical-align: text-bottom;
 }
 
 .breakpoint-exceptions-label {
-  padding-top: 2px;
+  line-height: 14px;
   padding-inline-start: 2px;
   padding-inline-end: 8px;
   cursor: default;
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
 
 .breakpoints-list .breakpoint,
```

Comment:
We seem to have the same selector at line 59, with 2px padding-top|bottom defined there. We should merge the two rulesets in one place.

Reply:
Fixed!